docs/enroute/Melbourne Centre/ASP.md

```diff
@@ -15,15 +15,11 @@
 | Bourke† | Melbourne Centre | 128.200 | ML-BKE_CTR |
 | Esperance† | Melbourne Centre | 123.950 | ML-ESP_CTR |
 
-† **Non-standard position** – may only be used in accordance with [VATPAC Ratings and Controller Positions Policy](https://cdn.vatpac.org/documents/policy/Controller+Positions+and+Ratings+Policy+v5.2.pdf)
+† *Non-standard positions* may only be used in accordance with [VATPAC Ratings and Controller Positions Policy](https://vatpac.org/publications/policies)
 
 ## Airspace
- 
-ASP is responsible for **ASP**, **ASW**, **WAR**, **FOR**, **WRA**, **BKE**, and **ESP** when they are offline.  
 
-When **AS TWR** is offline, AS CTR (Class D and C SFC-F125) within 80 DME AS reverts to Class G, and AS CTR (Class C F125-F245) within 80 DME AS reverts to Class E, and both are administered by ASP. Alternatively, ASP may provide a [top-down procedural service](../../../aerodromes/Alice) if they wish.
-=======
-When **AS TWR** is offline, AS CTR (Class D and C SFC-F125) within 80 DME AS reverts to Class G, and AS CTR (Class C F125-F245) within 80 DME AS reverts to Class E, and both are administered by ASP. Alternatively, ASP may provide a top-down procedural service if they wish.
+When **AS ADC** is offline, AS CTR (Class D and C `SFC` to `F125`) within 80 DME AS reverts to Class G, and AS CTR (Class C `F125` to `F245`) within 80 DME AS reverts to Class E, and both are administered by ASP. Alternatively, ASP may provide a [top-down procedural service](../../../aerodromes/Alice) if they wish.
 
 <figure markdown>
 ![Alice Springs Airspace](../assets/ASPENR.png){ width="700" }
```

Comment:
Did we still want to make the extending paragraph a warning? This & ISA, due to it being unique from other sectors?